scripts/common.bash

```diff
@@ -13,15 +13,21 @@ function for_each_unittest()
 	local check
 	local accel
 	local timeout
+	local run_under_efi
 	local rematch
 
+	run_under_efi=""
+	if [[ ${TEST_DIR} != ${TEST_SUBDIR} ]]; then
+		run_under_efi=true
```

Comment:
I think an alternative way to set `run_under_efi` is to read the `config.mak` (generated by `configure`) to check if the current project is configured to run under EFI. Its `TARGET_EFI` is set to `y` if configured with EFI, and an empty string if not.

But the current check is simple so I'm also good with this one.